src/builder/QuarkBuilder.sol

```diff
@@ -1167,7 +1197,14 @@ contract QuarkBuilder {
                 Actions.RepayActionContext memory cometRepayActionContext =
                     abi.decode(nonBridgeAction.actionContext, (Actions.RepayActionContext));
                 if (Strings.stringEqIgnoreCase(cometRepayActionContext.assetSymbol, paymentTokenSymbol)) {
-                    paymentTokenCost += cometRepayActionContext.amount;
```

Comment:
Unrelated to this line, but can you add a TODO on L1227 to test logic of wrap/unwrap affecting the `paymentTokenCost`? Basically, 🪄 made a good catch and we should have a test that breaks if the code is removed accidentally.

Reply:
added